cv32e40p/sim/uvmt/Makefile

```diff
@@ -48,5 +48,5 @@ MAKE_PATH            := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))
 export CORE_V_VERIF   = $(abspath $(MAKE_PATH)/../../..)
 export CV_CORE       ?= cv32e40p
 
-include ../Common.mk
+include ../ExternalRepos.mk
```

Comment:
While I didn't mind the "Common.mk" name, I have to agree that "ExternalRepos.mk" reflects the actual contents of that file much better.

Just to brainstorm a little bit, "RepoConfig.mk" or something like that might also work. But I think "ExternalRepos.mk" looks like the best suggestion.

Reply:
Almost anything is better than two included Makefiles with the same filename.